openedx/core/djangoapps/catalog/utils.py

```diff
@@ -231,11 +241,15 @@ def get_program_types(name=None):
     """
     user, catalog_integration = check_catalog_integration_and_get_user(error_message_field='Program types')
     if user:
-        api = create_catalog_api_client(user)
-        cache_key = f'{catalog_integration.CACHE_KEY}.program_types'
+        cache_key = f"{catalog_integration.CACHE_KEY}.program_types"
```

Comment:
Why do you change the single quotes with double quotes?

Reply:
reverted )